zookeeper-server/src/main/java/org/apache/zookeeper/cli/SyncCommand.java

```diff
@@ -74,7 +75,7 @@ public void processResult(int rc, String path, Object ctx) {
             if (resultCode == 0) {
                 out.println("Sync is OK");
             } else {
-                out.println("Sync has failed. rc=" + resultCode);
```

Comment:
Why aren't you handling explicitly the NoNode rc?

Reply:
@eolivelli Sorry about that. I will add catch block for NoNodeException and commit the changes shortly.